lib/internal/Magento/Framework/Mview/View/Changelog.php

```diff
@@ -173,7 +183,9 @@ public function getVersion()
         if (isset($row['Auto_increment'])) {
             return (int)$row['Auto_increment'] - 1;
         } else {
-            throw new \Exception("Table status for `{$changelogTableName}` is incorrect. Can`t fetch version id.");
+            throw new LocalizedException(
+                new Phrase("Table status for `{$changelogTableName}` is incorrect. Can`t fetch version id.")
```

Comment:
Please replace {$changelogTableName} with placeholder like in lib/internal/Magento/Framework/Mview/View/Changelog.php:104

Also, I would like to suggest use `\Magento\Framework\Exception\RuntimeException` instead of `\Magento\Framework\Exception\LocalizedException`.